domains/tanseer.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "username": "sheiktanseer"
+  },
+  "records": {
+    "CNAME": "sheiktanseer.github.io"
```

it seems you are using it to redirect to your github profile, use the URL instead
```suggestion
    "URL": "https://github.com/sheiktanseer"
```